### PR TITLE
setup AICoE-CI

### PR DESCRIPTION
**Pull request by goern**
Signed-off-by: Christoph Görn <goern@redhat.com>

### CODE REVIEW

File: .aicoe-ci.yaml [hunk 1]
```diff
@@ -1 +1,9 @@
-check: []
+check:
+  - thoth-build
+build:
+  build-stratergy: Containerfile
+  dockerfile-path: github-glue/Containerfile
```
Comment (harshad16) [inline .aicoe-ci.yaml]:
```suggestion
  dockerfile-path: Containerfile
```

**Review by harshad16 — COMMENTED** [review]
One change needed
rest looks good

File: .aicoe-ci.yaml (hunk 1 above)
Comment (goern) [reply .aicoe-ci.yaml]: it does not have to point to a specific containerfile?

**Comment by goern** [comment]
or.. well... maybe it should be on the subdirectory? if there is more components==directories to come?!

**Comment by goern** [comment]
@harshad16 will the repository on quay be created by aicoe-ci?

**Comment by harshad16** [comment]
> will the repository on quay be created by aicoe-ci?

Nope, admin has to create the repo.
I have create one for now. https://quay.io/repository/operate-first/bragi

**Comment by goern** [comment]
repository is available at https://quay.io/repository/operate-first/bragi?tab=info

**Pull request merged**